### PR TITLE
Fix cursor position of <C-o> command in insertmode

### PR DESCRIPTION
**Pull request by hy950831**
What real vim does.

What VscodeVim does currently.

What VscodeVim does after fix

to reproduce this bug. just type `i abcdef <Esc> hh i <C-O> l`
and i also updated the corresponding test case.

### CODE REVIEW

**Comment by jpoon** [comment]
From the travis logs, `Prettier Failed. Run `gulp` or `gulp forceprettier`.

File: package-lock.json [hunk 1]
```diff
@@ -1,5718 +0,0 @@
-{
```
Comment (jpoon) [inline package-lock.json]: Please un-delete this file.

**Comment by jpoon** [comment]
Thanks @hy950831

**Comment by hy950831** [comment]
@jpoon Sorry i didn't notice that you replied me, Thanks for merging.

**Pull request merged**